webpack/assets/javascripts/react_app/components/LoginPage/LoginPage.fixtures.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+export const adjustProps = ({ token, version }) => ({
+  container: {
+    backgroundUrl: '/assets/login_background.jpg',
```

Comment:
@ShimShtein will this break branding?

Reply:
It shouldn't. Branding knows how to replace images with branded ones. So as long as there is a branded version of `login_background.jpg` it should work.